hail/python/hail/experimental/plots.py

```diff
@@ -264,3 +269,148 @@ def scale_file_sizes(file_sizes):
         return Tabs(tabs=[Panel(child=entries_grid, title='Entries'), Panel(child=rows_grid, title='Rows')])
     else:
         return rows_grid
+
+def cnv_variant_cluster(mt, variant_range, show_plot=False, consecutive_interval=True):
+    """Create a (set of) scatterplot(s) showing SNP clusters per variant specified.
+
+    Parameters
+    ----------
+    mt : hail matrix table with locus, GTA, NORMX, NORMY, THETA values
+    variant_range : either string with interval ('1:1000-2000') or list of positions (can be 1)"
+    show_plot : default False, adivsed to only use this for small number of variants.
+    consecutive_interval : default True, if variant range is interval or list
+
+    Dependencies
+    -----------
+    Script requires : hail, bokeh.plotting, bokeh.io (), bokeh.models (ColumnDataSource, HoverTool), bokeh.models.tools
+
+    Returns
+    -------
+    :class:`bokeh.plotting.figure.Figure` OR a list of :class:`bokeh.plotting.figure.Figure`'s
+
+    """
+    def run_plotting(position, show_plot):
+        mt_variant = mt.filter_rows(['locus'] == hl.eval(hl.parse_locus(position)))
+        mt_variant_df = mt_variant.entries().to_pandas()
+
+        #Partition each genotype call
+        GT_HET = mt_variant_df.loc[mt_variant_df['GTA'] == '1/0']
+        GT_HOM_DOM = mt_variant_df.loc[mt_variant_df['GTA'] == '0/0']
+        GT_HOM_R = mt_variant_df.loc[mt_variant_df['GTA'] == '1/1']
+        GT_NA = mt_variant_df.loc[mt_variant_df['GTA'] == './.']
```

Comment:
should this be anything that is not `./.` or `./.` explicitly?

Reply:
I can double check with Tim, but all I ever found was ./. explicitly. I could also just use conditional statements.